### DISCUSSION TITLE
if the delete discord message deleted 4000 do i will get ban ?[Question]

### DISCUSSION THREAD

**Discussion by SL088** [victornpb/undiscord#274] [1. Questions & Answers]
> WARNING: Github Issues are publicly visible, do not post sensitive information here.
> You may want to use the "Screenshot mode" before taking screenshots or copying logs.

Describe what your question is, try to give as much detail as possible so I can help you.

Post screenshots if applicable.

**Comment by SL088** [comment]
and like two or more of useres i want to delete the direct message from me do i will get ban ?
like the messages i send it 2000 or more
the same time i do it

**Comment by victornpb** [comment]
Sorry it's unclear what the question is

**Comment by PatrickSantoZZ** [comment]
i understand the question. no you dont get a ban. i deleted over 20k messages in the last 2-3 days. so this is in the current state safe.

**Comment by ghost** [comment]
Can I delete messages from servers I have been banned from?

**Comment by ghost** [comment]
Is there in possibility there could be a patch or update to add a bypass API responded with 403 error missing access 50001  have multiple things deleted for servers you've been banned?

**Comment by ghost** [comment]
> Sorry it's unclear what the question is

Is there a way you can help me, I need a way to bypass API responded with 403 error missing access 50001 from a server I been banned in, Is there a way you could develop a patch or extension that can help overate access from those servers. I wouldn't mind paying you if you could solve my issue.

**Comment by ghost** [comment]
> There's probably not much you can do about it, AFAIK. Have you considered deleting your account and at least making your messages say they were written by "Deleted User#0000"?

I could but my messages have personal information pertaining to me, like photos of my face, my youtube channel, my first and last name. I'm kind being harassed for idiotic reason, I just want to get rid of my old stuff on this server that is related to me. there just things that I posted from late 2017 to mid 2018 and didn't thought anyone would bring them out just to make them look like memes and make fun about my mental health.

**Comment by ghost** [comment]
How do I get ahold of them? Throw trust&safety?
